scormxblock/static/js/src/scormxblock.js

```diff
@@ -128,34 +128,20 @@
             const data = getPackageData();
             data['name'] = name;
 
-            fetch(getValueUrl, {
-                method: 'POST',
-                headers: {
-                  'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8',
-                  'X-CSRFToken': GetCookie('csrftoken')
-                },
-                body: JSON.stringify(data),
-                credentials: 'same-origin',
-                keepalive: true
-            }).then(function(response) {
-                if (response.ok) {
-                    const content = response.json();
-                    if(content.error) {
-                        LearningTribes.Notification.Error({
-                            title: window.gettext("We're having trouble saving your work"),
-                            message: window.gettext(content.error),
-                        });
-                    } else {
-                        return content.value;
```

Comment:
- This is an async return. The async return cause function "GetValue()" returns "undefined".